atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SerializableTransaction.java

```diff
@@ -427,7 +427,7 @@ boolean isSerializableTable(TableReference table) {
         // If the metadata is null, we assume that the conflict handler is not SERIALIZABLE.
         // In that case the transaction will fail on commit if it has writes.
         ConflictHandler conflictHandler = conflictDetectionManager.get(table);
```

Comment:
Can we just fix this messed up API and have this return `Optional`? Especially since `ConflictDetectionManager` already does (though still `@Nullable` 🗡️

Reply:
yep, done!